agent/bench-scripts/pbench-uperf

```diff
@@ -402,36 +426,39 @@ if [ ! -z "$clients" -a "$postprocess_only" != "y" ]; then
 		error_log "    vsock-servers(${v_cnt}): ${vsock_servers}"
 		exit 1
 	fi
-fi
-# before we run a test, verify clients and servers are accessible and install
-# uperf if necessary
-if [ "$postprocess_only" != "y" ]; then
-	# Always install locally
-	# FIXME: why is this necessary?
-	check_install_rpm "${benchmark_rpm}" "${ver}" "${match}"
+
+	if [[ ${check_local} -gt 0 ]]; then
+		check_install_rpm "${benchmark_rpm}" "${ver}" "${match}"
+	fi
+
 	let err_cnt=0
 	err_clients=""
+	_clients=${clients:-localhost}
 	for client in ${clients//,/ }; do
-		debug_log "checking for uperf on client $client"
-		ssh $ssh_opts $client pbench-uperf --install
-		if [ $? -ne 0 ]; then
-			let err_cnt=err_cnt+1
-			err_clients="$err_clients $client"
+		if [[ "${client}" != "127.0.0.1" && "${client}" != "localhost" ]]; then
```

Comment:
You do this so often; and while it's relatively simple, I wonder if a `is_nonlocal` function would make the code just a bit easier to read and maintain? (If nothing else, it eliminates the remote chance of a typo in some obscure path that we'll miss.)

Reply:
And, are there other definitions of "local host" which we might need or want to consider?  (I think my container had almost six....)

Reply:
The recent changes resolve *my* comment, but not necessarily Webb's. Do we want to consider handling other definitions of "localhost" aside from `localhost` and `127.0.0.1`? E.g., `ip addr`, or `::`?? (The latter being especially difficult as there are an almost vast number of valid textual aliases that reduce to `::`, and `ip addr` for example reports it as the fully resolved `00:00:00:00:00:00`. Though IPv6 support is surely at best a PR for another day...)

Reply:
Yes, there are, but I'd rather not deal with that in this PR as I'd want to leverage a common function for that shared across various consumers in the agent code.